manifests/compute_node/config.pp

```diff
@@ -29,102 +29,78 @@
   $ohd_deb_repo      = $one::preseed_ohd_deb_repo,
   $debian_mirror_url = $one::preseed_debian_mirror_url,
   $preseed_tmpl      = $one::preseed_tmpl,
+  $libvirtd_cfg      = $one::libvirtd_cfg,
+  $libvirtd_source   = $one::libvirtd_source,
+  $libvirtd_srv      = $one::libvirtd_srv
 ){
 
-  File {
-    ensure => 'present',
-  }
-
   validate_string ($debian_mirror_url)
   validate_hash   ($preseed_data)
 
   file { '/etc/libvirt/libvirtd.conf':
-    owner  => 'root',
+    ensure => file,
     source => 'puppet:///modules/one/libvirtd.conf',
-    notify => Service[$one::libvirtd_srv],
-  }
+    notify => Service[$libvirtd_srv],
+  } ->
 
-  file { $one::libvirtd_cfg:
-    owner  => 'root',
-    source => $one::libvirtd_source,
-    notify => Service[$one::libvirtd_srv],
-  }
+  file { $libvirtd_cfg:
+    ensure => file,
+    source => $libvirtd_source,
+    notify => Service[$libvirtd_srv],
+  } ->
 
   file { '/etc/udev/rules.d/80-kvm.rules':
-    owner  => 'root',
+    ensure => file,
     source => 'puppet:///modules/one/udev-kvm-rules',
-  }
+  } ->
 
   file { '/etc/sudoers.d/10_oneadmin':
-    owner  => 'root',
-    group  => 'root',
-    mode   => '0644',
+    ensure => file,
     source => 'puppet:///modules/one/oneadmin_sudoers',
-  }
+  } ->
 
   file { '/etc/sudoers.d/20_imaginator':
-    owner  => 'root',
-    group  => 'root',
-    mode   => '0644',
+    ensure => file,
     source => 'puppet:///modules/one/sudoers_imaginator',
-  }
-
-  if $::osfamily == 'Debian' {
-    file { '/sbin/brctl':
-      ensure => link,
-      target => '/usr/sbin/brctl',
-    }
-  }
-
-  if $::osfamily == 'RedHat' and versioncmp($::operatingsystemmajrelease, '7') < 0  {
-    file { '/sbin/brctl':
-      ensure => link,
-      target => '/usr/sbin/brctl',
-    }
-  }
+  } ->
 
   file { 'polkit-opennebula':
+    ensure => file,
     path   => $::osfamily ? {
       'RedHat' => '/etc/polkit-1/localauthority/50-local.d/50-org.libvirt.unix.manage-opennebula.pkla',
       'Debian' => '/var/lib/polkit-1/localauthority/50-local.d/50-org.libvirt.unix.manage-opennebula.pkla',
     },
-    owner  => 'root',
-    group  => 'root',
-    mode   => '0644',
     source => 'puppet:///modules/one/50-org.libvirt.unix.manage-opennebula.pkla',
-  }
+  } ->
 
   file { '/etc/libvirt/qemu.conf':
-    owner  => 'root',
-    group  => 'root',
-    mode   => '0644',
+    ensure => file,
     source => 'puppet:///modules/one/qemu.conf'
-  }
+  } ->
 
-  # Imaginator
   file { '/var/lib/one/.virtinst':
-    ensure => directory,
-    owner  => 'oneadmin',
-    group  => 'oneadmin',
-    mode   => '0755',
```

Comment:
Please keep the mode in place.
You might get trouble when umask is set to a more restrictive setting.